roles/osm-vector-maps/defaults/main.yml

```
@@ -8,13 +8,17 @@
 # If nec, change them by editing /etc/iiab/local_vars.yml prior to installing!
 
 # The following soft coded variables allow testing, before pulling PR's into master
-# osm_repo_url: https://raw.githubusercontent.com/iiab/maps
-osm_repo_url: https://raw.githubusercontent.com/georgejhunt/maps
+#osm_repo_url: https://raw.githubusercontent.com/iiab/maps
 #maps_branch: 'master'    # Quotes not required
-maps_branch: 'maps7.3'
+
+#osm_repo_url: https://raw.githubusercontent.com/georgejhunt/maps
+#maps_branch: 'maps7.3'
+osm_repo_url: https://raw.githubusercontent.com/jvonau/maps
+maps_branch: github_host
```

Should these 2 lines be revised to...something like?

```suggestion
osm_repo_url: https://raw.githubusercontent.com/iiab/maps
maps_branch: master
```

---

Used in place of an override within local_vars, harder everybody to test if you have to edit anything. Those are only in place for testing this PR and should be changed before merging,

---

Yup!  Hopefully @georgejhunt can weigh in this morning if he has time (on both these PR's) to help finalize this PR here and also:

- PR iiab/maps#54

---

> Would need the last commit rolled back before committing to master, it's provided for testing convenience of the new github location scheme until the above changes are implemented in maps.